pages/common/chafa.md

```diff
@@ -0,0 +1,25 @@
+# chafa
+
+> Image printing in the terminal.
+> See also: `catimg` and `pixterm`.
+> More information: <https://hpjansson.org/chafa>.
+
+- Print an image or animated GIF to the terminal:
+
+`chafa {{path/to/input}}`
+
+- Use 24-bit [c]olor:
+
+`chafa -c full {{path/to/input}}`
+
+- Use dithering to improve output with small color palettes:
+
+`chafa -c 16 --dither ordered {{path/to/input}}`
+
+- Generate output like a low-resolution raster image:
```

Comment:
Why `generate` and not `print` here?

Reply:
Printing to terminal is just "spitting out" what it's generated. In the last two examples, I think it's a little more specific to say something like "generate" instead of the verb "use" I'd used before that. So basically, nothing about the printing process is being changed, whereas the process of generation *is*. And it's not simply using a one feature; the whole process is *generating* something similar to a raster image. I hope my reasoning makes sense.